ssh/src/extension.cpp

```diff
@@ -122,7 +122,7 @@ void Ssh::Extension::handleQuery(Query * query) const {
     else
     {
         // Check sanity of input
-        QRegularExpression re(R"raw(^(?:(\w+)@)?\[?((?:\w[\w:]*|[\w\.]*))\]?(?::(\d+))?$)raw");
+        QRegularExpression re(R"raw(^(?:(\w+)@)?\[?((?:\w[\w:-]*|[\w\.-]*))\]?(?::(\d+))?$)raw");
```

Comment:
Hmm, I'm not sure if I've misunderstood the changes in the PR but I can see hostnames with hyphens when I trigger the ssh plugin

![latest-screenshot](https://user-images.githubusercontent.com/5816719/78150335-12727f80-742f-11ea-9cd8-5963fa5f47aa.png)

Reply:
oh, I see, you're right!
Thanks for the explanation!